test/exercises/leap_test.rb

```diff
@@ -0,0 +1,60 @@
+require "test_helper"
+require 'pry'
```

Comment:
This does not seem to be used in this code, so likely was a debugging tool.  Our debugging tools probably should not end up being exposed.

Consider using `ruby -rpry` and Ruby will alert you loudly if you forget any of the debugging things in the code, which provides an additional advantage!

Reply:
I removed the require statement, which I actually copied from the one of the other test files. Thanks for the tip.

Reply:
This could be another "chore" that should be done.  It does reveal that pry is involved, and it is definitely useful, but it is confusing when you see it and it does nothing but load extra code into memory, with nothing to do.

But we can do this repository while during that chore.